packages/gatsby-source-contentful/src/__tests__/__snapshots__/normalize.js.snap

```diff
@@ -1849,10 +1851,14 @@ Our Lemnos products are made carefully by our craftsmen finely honed skillful te
       },
     },
   ],
-  Array [],
   Array [
     Object {
       "fields": Object {
+        "jsonStringTest": Object {
+          "en-US": Array [
+            "test",
+          ],
+        },
```

Comment:
@pieh i guess we can simply get rid of the field in ctf and reexport?

Reply:
Yeah, this might be good opportunity to clean up "testing" stuff from the space - I remember adding some of those when debugging something and then I didn't clean it up

Reply:
+1!

Reply:
We should keep this as it tests if arrays of strings are properly converted to json graphql type